### ISSUE TITLE
[FXA-Settings] Footer typo

### ISSUE THREAD

**Issue by janiceahn** [mozilla/fxa#7760]
## Description

Web Site --> Website

### Steps to reproduce

![image](https://user-images.githubusercontent.com/6219141/110514016-d7fc7280-80d4-11eb-8e63-092ca9ac97ed.png)

### What should happen
<img width="359" alt="Untitled 3" src="https://user-images.githubusercontent.com/6219141/110514081-ed719c80-80d4-11eb-8fae-201374b657d7.png">

**Comment by ShrutiKatpara** [comment]
hey, is it done? Or shall I move ahead and fix it?

**Comment by LZoog** [comment]
@kshru9 Feel free to grab this if you'd still like to work on it! :) 

This footer can be found on the Settings page.

**Comment by karinabk** [comment]
if it is still open, I am interested in this issue

**Comment by Deek9399** [comment]
Hello @LZoog, If this issue is available, I would like to work on it.

**Comment by jodyheavener** [comment]
@Jane-Kotovich @karinabk @Deek9399 Looks like you all currently have issues assigned, so let's space it out and if your issue gets closed feel free to check back in here and I'll be happy to assign you! 🙂

**Comment by dhruva248** [comment]
Hi, I am an Outreachy applicant and interested to work on this issue, I am currently not assigned any issue! Can I work on it?

**Comment by LZoog** [comment]
@dhruva248 you probably commented here before being assigned to another but let's wait until the other issue you've now got assigned to you is fixed! 😄 Thanks!

**Comment by dhruva248** [comment]
> @dhruva248 you probably commented here before being assigned to another but let's wait until the other issue you've now got assigned to you is fixed! 😄 Thanks!
Yes @LZoog, i commented here prior to being assigned another one! :)

**Comment by dhanesh12twice** [comment]
Hi, I am an Outreachy applicant and  I still dont have any issue to fix. Can I take this issue to fix? @LZoog

**Comment by Jane-Kotovich** [comment]
Heya! Im an Outreachy applicant and I would love to take that issue to fix if possible! Thanks!

**Comment by Deek9399** [comment]
Hi @LZoog, Can I work on this issue?

**Comment by Jane-Kotovich** [comment]
Hey @jodyheavener same as before :) Im an Outreachy applicant and could I please grab this as my second issue? Thanks!

**Comment by jodyheavener** [comment]
Sorry all for the delays on this thread everyone! @dhanesh12twice if you're still interested in this issue it's all yours :)

**Comment by Jo-bot86** [comment]
Hey @jodyheavener, if no one is interested in it anymore, I would be happy to work on it.

**Comment by LZoog** [comment]
Hey @dhanesh12twice, just following up here. Are you currently working on this? 🙂

**Comment by dhanesh12twice** [comment]
@LZoog apologies for the delay , ive some health issue, now it is okay. So yeah , i will work and report you within two days. Thanks

**Comment by NegiAkash890** [comment]
Hi, can I fix this ?

**Comment by LZoog** [comment]
Hi @NegiAkash890, thank you for your interest! Let's give @dhanesh12twice another few days, I don't want to pull the rug out from under them here, but if they don't have a PR up for it by the end of the week I'd say go ahead if you'd like to then. Please feel free to work on any other good first issues without an assignee if you'd like :)

**Comment by Deek9399** [comment]
Hey, @LZoog Can I try this, if @dhanesh12twice and @NegiAkash890 are working on something else?
